src/xdocs/config_modifier.xml

```diff
@@ -22,6 +22,206 @@
+          </tr>
+          <tr>
+            <td>violateImpliedPublicField</td>
+            <td>
+              Control whether to enforce that <code>public</code> is not being implied by the
+              compiler on interface fields (true).</td>
+            <td><a href="property_types.html#boolean">Boolean</a></td>
+            <td><code>true</code></td>
+            <td>8.11</td>
+          </tr>
+          <tr>
+            <td>violateImpliedStaticField</td>
+            <td>
+              Control whether to enforce that <code>static</code> is not being implied by the
+              compiler on interface fields (true).</td>
+            <td><a href="property_types.html#boolean">Boolean</a></td>
+            <td><code>true</code></td>
+            <td>8.11</td>
+          </tr>
+          <tr>
+            <td>violateImpliedFinalField</td>
+            <td>
+              Control whether to enforce that <code>final</code> is not being implied by the
+              compiler on interface fields (true).</td>
+            <td><a href="property_types.html#boolean">Boolean</a></td>
+            <td><code>true</code></td>
+            <td>8.11</td>
+          </tr>
+          <tr>
+            <td>violateImpliedPublicMethod</td>
+            <td>
+              Control whether to enforce that <code>public</code> is not being implied by the
+              compiler on interface methods (true).</td>
+            <td><a href="property_types.html#boolean">Boolean</a></td>
+            <td><code>true</code></td>
+            <td>8.11</td>
+          </tr>
+          <tr>
+            <td>violateImpliedAbstractMethod</td>
+            <td>
+              Control whether to enforce that <code>abstract</code> is not being implied by the
+              compiler on interface methods (true).</td>
+            <td><a href="property_types.html#boolean">Boolean</a></td>
+            <td><code>true</code></td>
+            <td>8.11</td>
+          </tr>
+          <tr>
+            <td>violateImpliedPublicNested</td>
+            <td>
+              Control whether to enforce that <code>public</code> is not being implied by the
+              compiler on interface nested types (true).</td>
+            <td><a href="property_types.html#boolean">Boolean</a></td>
+            <td><code>true</code></td>
+            <td>8.11</td>
+          </tr>
+          <tr>
+            <td>violateImpliedStaticNested</td>
+            <td>
+              Control whether to enforce that <code>static</code> is not being implied by the
+              compiler on interface nested types (true).</td>
+            <td><a href="property_types.html#boolean">Boolean</a></td>
+            <td><code>true</code></td>
+            <td>8.11</td>
+          </tr>
+        </table>
+      </subsection>
+
+      <subsection name="Examples">
+        <p> To configure the check: </p>
+        <source>
+&lt;module name=&quot;InterfaceMemberImpliedModifier&quot;/&gt;
+        </source>
```

Comment:
please add here small snipped of code to show what behavior will be (with trailing comments "//violation")

Reply:
Done